lib/presentation/view/custom_list_tiles/shared_space_member_list_tile_builder.dart

```diff
@@ -29,66 +29,90 @@
-      contentPadding: EdgeInsets.only(left: 24, top: 10, bottom: 10),
+      contentPadding: EdgeInsets.only(left: 24, top: 10, bottom: 10, right: 20),
       leading: LabelAvatarBuilder(_name.characters.first.toUpperCase())
-          .key(Key('label_shared_space_member_avatar'))
-          .build(),
+        .key(Key('label_shared_space_member_avatar'))
+        .build(),
+      trailing: allowDelete
+        ? GestureDetector(
+          onTap: () {
+            if (onDeleteMemberCallback != null) {
+              onDeleteMemberCallback.call();
+            }
+          },
+          child: Icon(Icons.close, color: AppColor.deleteMemberIconColor))
+        : SizedBox.shrink(),
       title: Column(crossAxisAlignment: CrossAxisAlignment.start, children: [
         Padding(
-            padding: EdgeInsets.only(bottom: 8),
-            child: Text(_name,
-                style: TextStyle(
-                    fontSize: 14,
-                    fontWeight: FontWeight.w500,
-                    fontStyle: FontStyle.normal,
-                    color: AppColor.loginTextFieldTextColor))),
-        Text(_email,
+          padding: EdgeInsets.only(bottom: 8),
+          child: Text(_name,
             style: TextStyle(
-                fontSize: 14,
-                fontWeight: FontWeight.normal,
-                fontStyle: FontStyle.italic,
-                color: AppColor.documentModifiedDateItemTextColor)),
+              fontSize: 14,
+              fontWeight: FontWeight.w500,
+              fontStyle: FontStyle.normal,
+              color: AppColor.loginTextFieldTextColor))),
+        Text(_email,
+          style: TextStyle(
+            fontSize: 14,
+            fontWeight: FontWeight.normal,
+            fontStyle: FontStyle.italic,
+            color: AppColor.documentModifiedDateItemTextColor)),
         Padding(
-            padding: EdgeInsets.only(top: 8),
-            child: GestureDetector(
-              onTap: () {
-                if(onSelectedRoleCallback != null) {
-                  onSelectedRoleCallback.call();
-                }
-              },
-              child: Row(
-                mainAxisSize: MainAxisSize.min,
-                children: [
-                  Text(_roleName,
-                      style: TextStyle(
-                          fontSize: 14,
-                          fontWeight: FontWeight.normal,
-                          fontStyle: FontStyle.normal,
-                          color: AppColor.workgroupNodesSurfingBackTitleColor)),
-                  Icon(Icons.arrow_drop_down, color: AppColor.primaryColor)
+          padding: EdgeInsets.only(top: 8),
+          child: GestureDetector(
+            onTap: () {
+              if(onSelectedRoleCallback != null && allowUpdate) {
+                onSelectedRoleCallback.call();
+              }
+            },
+            child: Row(
+              mainAxisSize: MainAxisSize.min,
+              children: [
+                Text(_roleName,
```

Comment:
how about when click to this `Row` if `allowUpdate == false`

Reply:
it's clickable only when `allowUpdate == true`:

```
if(onSelectedRoleCallback != null && allowUpdate) {
	onSelectedRoleCallback.call();
}
```